test/commute.js

```diff
@@ -17,7 +16,26 @@ describe('commute', function() {
   });
 
   it('works on Algebraic Data Types such as "Maybe"', function() {
```

Comment:
Note that you didn't change the name, `Maybe` -> `Identity`.

But are there compelling reasons to not allow a test to require an implementation of `Maybe` -- I don't care if its that one, the live one from ramda-fantasy, or from Folktale?

I know that you prefer unit tests to have no external references, but this seems to be pushing it too far. Should we also implement our own mini Promise library just to avoid an include in a test file?

I also personally prefer the semantics of `Maybe`, as people actually understand that one for the most part, and recognize that it, in fact, does something useful.  Using `Identity` feels as though it doesn't really demonstrate anything.